src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Types/AggregateType.cs

```diff
@@ -29,8 +29,8 @@ internal partial class AggregateType : CType
         public short typeConverterID;
 #endif // !CSEE
 
-        public bool fConstraintsChecked;    // Have theraints been checked yet?
-        public bool fConstraintError;       // Did theraints check produce an error?
+        public bool fConstraintsChecked;    // Have the constraints been checked yet?
+        public bool fConstraintError;       // Did the constraints check produce an error?
```

Comment:
It is interesting... I wonder in what situation in the past someone did a search and replace to remove all "const".